package/crds/accesscontextmanager.gcp.upbound.io_serviceperimeterresources.yaml

```diff
@@ -35,8 +35,7 @@ spec:
     schema:
       openAPIV3Schema:
         description: ServicePerimeterResource is the Schema for the ServicePerimeterResources
-          API. Allows configuring a single GCP resource that should be inside of a
-          service perimeter.
+          API. Allows configuring a single GCP resource that should be inside the
```

Comment:
Turns out code generation wasn't able to properly handle an HTML code tag in [the documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/access_context_manager_service_perimeter_resource). IMHO, this is not crucial, but let's be aware of the issue.

Reply:
I fixed the description by adding a manual configuration, but I agree that these should be handled during code generation: [c75257f](https://github.com/crossplane-contrib/provider-upjet-gcp/pull/553/commits/c75257f49e188cf568dbcd548f1883388dbb5faf)[c75257f](https://github.com/crossplane-contrib/provider-upjet-gcp/pull/553/commits/c75257f49e188cf568dbcd548f1883388dbb5faf)[c75257f](https://github.com/crossplane-contrib/provider-upjet-gcp/pull/553/commits/c75257f49e188cf568dbcd548f1883388dbb5faf)